static/src/javascripts/projects/common/modules/experiments/tests/acquisitions-epic-recurring-contribution-uk-support-proposition.js

```diff
@@ -13,15 +13,15 @@ export const acquisitionsEpicRecurringContributionUkSupportProposition = makeABT
         id: 'AcquisitionsEpicRecurringContributionUkSupportProposition',
         campaignId: 'sandc_epic_recurring_contribution_uk_support_proposition',
 
-        start: '2017-08-07',
+        start: '2017-08-14',
         expiry: '2017-09-13',
 
         author: 'svillafe',
         description:
             'This creates a two button version of the epic that links off to the new support frontend contribution landing page',
         successMeasure: 'Annualised value',
         idealOutcome:
-            'We see an uplift in annualised value when recurring contributions are available.',
+            'We are going to test if the new  stacks correcntly handle the different flows.',
 
         audienceCriteria: 'UK all devices',
         audience: 1,
```

Comment:
We need to check this number - as things stand I'm pretty sure this would mean half the audience go to support.theguardian.com while half end up in the pay-in-epic test (of which a third, i.e. a sixth of the total, would  see the normal epic). Seems like we might want to leave more people on the normal epic